### DISCUSSION TITLE
Problem at Upload Feeds with php 'SignatureDoesNotMatch'

### DISCUSSION THREAD

**Discussion by KemnaMOL** [amzn/selling-partner-api-models#3666] [General]
I try to upload the json-file to to received URL.
But I ever get the error SignatureDoesNotMatch.
Often the problem is at the content-type.
I start the feed with
```
{
    "contentType":"application/json; charset=UTF-8"
	}
```
later I try to upload with curl:
```
	$ch = curl_init();
        curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'PUT');
	curl_setopt($ch, CURLOPT_URL, $url);
        curl_setopt($ch, CURLOPT_HTTPHEADER, 'content-type:application/json; charset=UTF-8');
        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
        curl_setopt($ch, CURLOPT_POSTFIELDS, $post);
	$out = curl_exec($ch);
```
I see a different in contentType vs content-type. Can that be the problem?

**Comment by salamichel** [comment]
Hi, how did you solved your issue ? i experience the same problem when uploading feed

**Reply by albinsopaj** [reply]
Any luck so far? Did you manage to solve it?
